Braintree.podspec

```diff
@@ -24,6 +24,7 @@ Pod::Spec.new do |s|
   s.subspec "AmericanExpress" do |s|
     s.source_files  = "Sources/BraintreeAmericanExpress/*.swift"
     s.dependency "Braintree/Core"
+    s.resource_bundle = { "BraintreeAmericanExpress_PrivacyInfo" => "Sources/BraintreeAmericanExpress/PrivacyInfo.xcprivacy" }
```

Comment:
We can name this whatever we want, but it has to be unique per module.

Reply:
It's verbose but I think makes sense considering we need 1 manifest per module to follow this pattern